src/quo/components/text_input.cljs

```diff
@@ -205,7 +207,7 @@
            (when before
              [accessory-element before])
            [rn/text-input
-            (merge {:style                   (text-input-style multiline input-style monospace before after)
+            (merge {:style                   (text-input-style multiline input-style monospace before after new-ui?)
```

Comment:
I believe we have agreed to stop using new-ui prop in this manner. @flexsurfer What's the best approach here?
the ui2 namespace only seems to solve this issue for everything in 
src/status_im/ui

Should we start copying some code into quo2 as well?

Reply:
I think its okay to move this file to quo2. It will basically be just a copy but its what we agreed

Reply:
yes we can just move it